classes/form/FormField.php

```diff
@@ -181,4 +183,16 @@ public function getConstraints()
     {
         return $this->constraints;
     }
+
+    public function setAutocomplete($autocomplete)
```

Comment:
```suggestion
    public function setAutocompleteAttribute($autocomplete)
```

Suggestion: be as accurate as possible in function name to make sure people understand what it is. With 'autocomplete' alone someone could think it's, for example, a feature that performs ajax queries to auto-complete what you type 😄

Reply:
Also @Progi1984 will tell you to type-hint this if I am not wrong 😛